modules/ui.js

```diff
@@ -0,0 +1,145 @@
+  }
+}
+document.addEventListener('DOMContentLoaded', UI.displayBooks);
+
+document.addEventListener('DOMContentLoaded', () => {
+  const dateP = document.querySelector('.date');
+  const currentDate = DateTime.now();
+  dateP.textContent = `${currentDate.toLocaleString(DateTime.DATETIME_MED)}`;
+});
+
+let isbn = document.querySelector('#isbn').value;
+isbn = '0';
+// Event: Add a book.
+document.querySelector('#book-form').addEventListener('submit', (e) => {
+  // Prevent submit default event
+  e.preventDefault();
+  // get form values
+  const title = document.querySelector('#title').value;
+  const author = document.querySelector('#author').value;
+  // let isbn = document.querySelector('#isbn');
+  isbn += 1;
+
+  // Validate
+  if (title === '' || author === '' || isbn === '') {
+    UI.showAlert('Please fill in all fields', 'emptyfields');
+  } else {
+    // instantiate book
+    const book = new Book(title, author, isbn);
+
+    // Add book to UI
+    UI.addBookToList(book);
+
+    // Add book to localstorage
+
+    Store.addBook(book);
+
+    // Show success message
+    UI.showAlert('Book Added', 'success');
+
+    // Clear fields
+    UI.clearFields();
+  }
+});
+
+const listLink = document.querySelector('.list-link');
+const addNewLink = document.querySelector('.add-new-link');
+const contactLink = document.querySelector('.contact-link');
+
+const addNew = document.querySelector('.add-new');
+const listSection = document.querySelector('.list');
+const contactSection = document.querySelector('.contact');
+
+addNewLink.addEventListener('click', () => {
+  if (addNew.classList.contains('hidden')) {
+    addNew.classList.remove('hidden');
+    if (!listSection.classList.contains('hidden')) {
+      listSection.classList.add('hidden');
+    }
+    if (!contactSection.classList.contains('hidden')) {
+      contactSection.classList.add('hidden');
+    }
+  }
+});
+
+listLink.addEventListener('click', () => {
+  if (listSection.classList.contains('hidden')) {
+    listSection.classList.remove('hidden');
+    if (!addNew.classList.contains('hidden')) {
+      addNew.classList.add('hidden');
+    }
+    if (!contactSection.classList.contains('hidden')) {
+      contactSection.classList.add('hidden');
+    }
+  }
+});
+
+contactLink.addEventListener('click', () => {
+  if (contactSection.classList.contains('hidden')) {
+    contactSection.classList.remove('hidden');
+    if (!listSection.classList.contains('hidden')) {
+      listSection.classList.add('hidden');
+    }
+    if (!addNew.classList.contains('hidden')) {
+      addNew.classList.add('hidden');
+    }
+  }
+});
+
+// document.querySelector('#books-list').addEventListener('click', (e) => {
+//   //remove book from UI
+//   UI.deleteBook(e.target);
+
+//   //Remove book from localstorage
+//   Store.removeBook(e.target.parentElement.previousElementSibling.textContent);
+//   //Show success message
+//   UI.showAlert('Book Removed', 'success');
+// });
```

Comment:
- [ ] [Optional] Kindly remove this commented-out code from this file. Removing such makes your code look clean. 👍